stable/osg-hosted-ce/osg-hosted-ce/templates/deployment.yaml

```diff
@@ -173,13 +170,11 @@ spec:
         - name: log-volume
           mountPath: /var/log/condor-ce
         {{ end }}
-        {{ if .Values.BoscoOverrides.Enabled }}
-        {{ if .Values.BoscoOverrides.GitKeySecret }}
+        {{ if .Values.BoscoOverrides.RepoNeedsPrivKey }}
         - name: osg-hosted-ce-{{ .Values.Instance }}-gitkey
           mountPath: /etc/osg/git.key
           subPath: git.key
         {{ end }}
-        {{ end }}
```

Comment:
This appears to partially revert https://github.com/slateci/slate-catalog/commit/d367601857d8e7461f3c89c29c92cf02b790dc90#diff-3bb7f07a3f0bc43fe03b034bdcc8a765bbcb1fdb65a2afcaf0fbc61d53c49176. Is that intended?